spec/end_to_end_spec.js

```diff
@@ -15,11 +15,14 @@ describe('End to End Test', function() {
     getSecretFromSecretManagerStub = sinon.stub(getSecretFromSecretManager, 'getSecretFromSecretManager');
   });
 
-  beforeEach(function() {
+  beforeEach(async function() {
```

Comment:
afaik, `done` exists in `beforeEach as well
`beforeEach(async function(done) { done()`